canonical-web-and-design/react-components
# Remove custom css

## Done

- Removed custom CSS from MainTable and Modal components
- Update Vanilla to release candidate version v2.28.0-rc1

## Fixes

Fixes #155 
Fixes #411

## QA

### QA steps

- Check Modal docs, make sure it renders correctly: https://react-components-425.demos.haus/?path=/story/modal--default-story
- Check MainTable examples (empty table at the bottom), make sure it renders correctly: https://react-components-425.demos.haus/?path=/docs/maintable--empty


### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```
